### PR TITLE
Moving from feature/msbuild to dev

### PR DESCRIPTION
**Pull request by prafullbhosale**
Want to do a sanity check and get other eyes on what I have done here 😃 

`git checkout feature/msbuild`
`git merge dev --strategy=ours`
`git checkout -b pb/msbuild`

Did a binary comparison of the contents on disk of `pb/msbuild` and remote's `feature/msbuild`
only difference is in the .git folder.

cc @pranavkm  @natemcmaster

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ mono:
 os:
   - linux
   - osx
-osx_image: xcode7.3
+osx_image: xcode7.1
```
Comment (pranavkm) [inline .travis.yml]: Is this intentional?

Comment (prafullbhosale) [inline .travis.yml]: I don't think so. 
This must be because of the `--strategy=ours`

File: NuGet.config [hunk 2]
```diff
@@ -1,7 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <clear />
+    <add key="dotnet-cli" value="https://dotnet.myget.org/F/dotnet-cli/api/v3/index.json" />
+    <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json" />
```
Comment (natemcmaster) [inline NuGet.config]: This can be aspnetcore-ci-dev now

File: NuGetPackageVerifier.json [hunk 3]
```diff
@@ -1,7 +1,30 @@
 {
-    "Default": {
+    "adx": { // Packages written by the ADX team and that ship on NuGet.org
```
Comment (natemcmaster) [inline NuGetPackageVerifier.json]: Nit: remove comments

File: global.json [hunk 4]
```diff
@@ -1,9 +0,0 @@
-{
-  "projects": [
-    "src",
-    "test/TestApps"
-  ],
-  "sdk": {
-    "version": "1.0.0-preview2-1-003180"
```
Comment (natemcmaster) [inline global.json]: Actually, I'm about to re-add global.json to our converted repos with sdk:version set to 1.0.0-preview4-004233. It's gonna take a week or so to move everyone up to RC3 and I don't want to break RC2 repos in the meantime.

**Comment by prafullbhosale** [comment]
:up: :date: Reapplied changes from dev. The following are not yet reapplied: 
- 0dc42842df5d9c95a25c4bb270648e7d9a3ce46b, 
- 51f1c0af55b7d77d4c3d11eb72e761c6e77b219c 
- 02bb35de6f29db3ba3970e06bdad7f95c2d1d3ae. 
Will add these later on.

File: src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Microsoft.VisualStudio.Web.CodeGeneration.Core.csproj [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="1.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
+    <Reference Include="System" />
```
Comment (natemcmaster) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Microsoft.VisualStudio.Web.CodeGeneration.Core.csproj]: I believe these can be removed.

**Comment by pranavkm** [comment]
:shipit: as long as it compiles 👍

File: src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Properties/AssemblyInfo.cs [hunk 6]
```diff
@@ -5,10 +5,6 @@
 using System.Resources;
 using System.Runtime.CompilerServices;
 
+[assembly: AssemblyMetadata("Serviceable","true")]
```
Comment (natemcmaster) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Properties/AssemblyInfo.cs]: Not required. Internal.AspNetCore.Sdk will generate this.

File: global.json (hunk 4 above)
Comment (prafullbhosale) [inline global.json]: Added this back.

**Comment by prafullbhosale** [comment]
:up: :date:
@natemcmaster

**Pull request merged**